src/server/server_family.cc

```diff
@@ -2419,13 +2420,15 @@ void ServerFamily::Info(CmdArgList args, Transaction* tx, SinkReplyBuilder* buil
           current_snap_keys = res.current_keys;
           total_snap_keys = res.total_keys;
           perc = (static_cast<double>(current_snap_keys) / total_snap_keys) * 100;
+          big_value_preemptions = res.big_value_preemptions;
```

Comment:
I was thinking on a global counter similar to rdb_save_count
so that we can check this value even if snapshot is not in progress
use it in our py tests, check it is bigger than 0